cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/EquipmentExport.java

```diff
@@ -35,92 +35,95 @@ public final class EquipmentExport {
     private static final String RATIO_TAP_CHANGER_REGULATION_MODE_VOLTAGE = "voltage";
     private static final Logger LOG = LoggerFactory.getLogger(EquipmentExport.class);
 
-    public static void write(Network network, XMLStreamWriter writer) {
-        write(network, writer, new CgmesExportContext(network));
-    }
-
-    public static void write(Network network, XMLStreamWriter writer, CgmesExportContext context) {
+    private final String euNamespace;
+    private final String limitValueAttributeName;
+    private final String limitTypeAttributeName;
+    private final String limitKindClassName;
+    private final boolean writeInfiniteDuration;
+    private final boolean writeInitialP;
+
+    private final Map<String, String> mapNodeKey2NodeId = new HashMap<>();
+    private final Map<Terminal, String> mapTerminal2Id = new HashMap<>();
+    private final Set<String> regulatingControlsWritten = new HashSet<>();
+
+    EquipmentExport(CgmesExportContext context, XMLStreamWriter xmlWriter) {
+        super(context, xmlWriter);
+        euNamespace = context.getCim().getEuNamespace();
+        limitValueAttributeName = context.getCim().getLimitValueAttributeName();
+        limitTypeAttributeName = context.getCim().getLimitTypeAttributeName();
+        limitKindClassName = context.getCim().getLimitKindClassName();
+        writeInfiniteDuration = context.getCim().writeLimitInfiniteDuration();
+        writeInitialP = context.getCim().writeGeneratingUnitInitialP();
```

Comment:
I don't know if you need to store all these if they are already in the context which is stored.